src/components/SideBar/SideBar.tsx

```diff
@@ -75,16 +98,17 @@ const SideBar = () => {
 				</div>
 
 				<div className={styles.Footer}>
-					<PriceWidget isRoot={isRoot} />
-					<div className={cx(styles.ZeroIconContainer, { Hidden: isRoot })}>
-						<a
-							className={styles.Zero}
-							target="_blank"
-							href={URLS.ZERO}
-							rel="noreferrer"
-						>
-							<img alt={ALT_TEXT.ZERO_LOGO} src={DOMAIN_LOGOS.ZERO} />
-						</a>
+					<PriceWidget isRoot={!zna.startsWith('wilder')} />
+					<div
+						className={cx(styles.ZeroIconContainer, {
+							Hidden: isDefaultNetworkRootPath,
+						})}
+					>
+						{zna.startsWith('wilder') && (
```

Comment:
Ah I can see why you've removed the zero.tech link as we'd be displaying two zero logos in the sidebar which would look a bit weird. 

This does change the behaviour though, if a user clicks into 'tester' network, the zero logo won't take them to the zero root page, so this could be a bit misleading. Something we can pick up in due course